community/extensions.txt

```diff
@@ -0,0 +1,29 @@
+KISS EXTENSIONS
+________________________________________________________________________________
+
+Anything in the user's '$PATH' which matches the glob 'kiss-*' will interpreted 
+as an extension and be directly usable via the package manager. Here are some
+extensions written by the community.
+
+kiss-bin: an extension for dealing with binary package repositories
+________________________________________________________________________________
+
+Source: $/CarbsLinux/kiss-bin
+
+kiss-find: search for packages in all known kiss repositories
+________________________________________________________________________________
+
+Source: $/Admicos/kiss-repo/tree/master/custom/kiss-find
+
+
+kiss-rmdeps: Recursive package dependency removal tool
+________________________________________________________________________________
+
+Source: https://gist.github.com/FriendlyNeighborhoodShane/41593680b39c0c04cd82b5497ca25a26
+
+
+kiss-steal: Copy packages from any KISS repo automatically
+________________________________________________________________________________
+
+Source: $/Admicos/kiss-repo/tree/master/custom/kiss-steal
+
```

Comment:
I think it's best for it to be formatted in this way:

```
kiss-steal
________________________________________________________________________________

Copy packages from any KISS repository automatically.

Source: $/Admicos/kiss-repo/tree/master/custom/kiss-steal
```

Reply:
I updated the formatting.